shell/platform/windows/platform_handler_win32_unittests.cc

```diff
@@ -32,10 +32,47 @@ static constexpr int kArbitraryErrorCode = 1;
 
 }  // namespace
 
+// A test version of system clipboard.
+class MockSystemClipboard {
```

Comment:
Apparently I forgot to save a comment; trying again:

Is there a reason you are subclassing instead of using a `friend` test fixture?

Reply:
I didn't know the feature. I have no reason to avoid it.

Reply:
The FRIEND_TEST seems to require both target class and tests are in same namespace.
I also tried `friend class testing::PlatformHandlerWin32_ReleaseClipboard_Test;`, but it makes `error: use of undeclared identifier 'testing'`. Are there any macro which exists only on testing?

Reply:
I don't know that we've ever explicitly documented _not_ to do this in the engine, but I'll be honest; I had no idea `FRIEND_TEST` existed as part of googletest.

The namespacing issue is unfortunate. I wonder if we should just fix this across the board in our tests. A `testing` namespace is certainly useful for utility code to be used in tests, but not sure why we've elected to place tests themselves in it. I'll file a separate issue for discussion if we don't have one already.